lapce-ui/src/tab.rs

```diff
@@ -945,7 +945,24 @@ impl LapceTab {
                     LapceUICommand::VoltInstalled(volt) => {
                         let plugin = Arc::make_mut(&mut data.plugin);
                         plugin.installed.insert(volt.id(), volt.clone());
-                    }
+
+                        // if there is a value inside the installing map, remove it from there as soon as it is installed.
+                        if plugin.installing.contains_key(&volt.id()) {
+                            plugin.installing.remove(&volt.id());
```

Comment:
I don't think you need to check existence before removing an element from an IndexMap.

Reply:
yes you are right, I will change that!